cocotb/__init__.py

```diff
@@ -241,6 +241,7 @@ def _initialise_testbench(argv_):
         RANDOM_SEED = int(RANDOM_SEED)
         log.info("Seeding Python random module with supplied seed %d" % (RANDOM_SEED))
```

Comment:
```suggestion
        log.info("Seeding cocotb's Python and GPI parts with supplied seed %d" % (RANDOM_SEED))
```
?

Reply:
I changed the log message here to reflect source of the seed.